packages/os/bootstrap-containers-tmpfiles.conf

```
@@ -0,0 +1,3 @@
+d /etc/bootstrap-containers 0755 root root -
```

Do we need to have world readable bits set on this directory? Can we instead restrict it to privileged users?

```suggestion
d /etc/bootstrap-containers 0750 root root -
```

---

I expect it'd be fine to change this, though we should change `host-containers-tmpfiles.conf` to match if we do.

---

Fixed